ifrs_report/view/ifrs_view.xml

```diff
@@ -418,5 +418,9 @@
             action="action_ifrs_ifrs_form"
         />
 
+        <record id="account_group_auditory.group_account_user_audit" model="res.groups">
```

Comment:
@hbto In this sección were addes the menu_acces to the group on account_group_auditory